recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: conda-{{ version }}.tar.gz
   url: https://github.com/conda/conda/archive/{{ version }}.tar.gz
-  sha256: 1843355ccb93afb05f2a307e1fc37403fb9976da799236eebc3adee1c716c5fc
+  sha256: c6607fee920aae5d96528669fc9e61b9beff6cf8c634d1d85bc6f409d5146b44
 
 build:
   number: 0
```

Comment:
We should add `skip: True  # [py<36]`, https://github.com/conda/conda/blob/fc81d4217f6c1804e983d1698f8b7cc5bc01874c/setup.py#L85

Reply:
Good catch!

Reply:
Updated with @skupr-anaconda's comment.